lib/makews.js

```diff
@@ -0,0 +1,49 @@
+const core = require("@actions/core");
+const exec = require("@actions/exec")
+
+async function makews(createworkspace, workspace, execfile) {
+    try {
+        let stdout = '';
+        let stderr = '';
+
+        const options = {};
+        options.listeners = {
+            stdout: (data) => {
+                stdout = data.toString();
+            },
+            stderr: (data) => {
+                stderr = data.toString();
+            }
+        };
+        options.cwd = './';
+        await exec.exec(execfile, [`workspace`, `list`], options);
+
+        if (createworkspace != "false") {
+            if (matchExact(new RegExp(`^${workspace}$`, 'gm'), stdout)) {
+                core.info(`workspace does not exists, creating as create-workspace param not set to false`)
```

Comment:
Nice! I think this will be very helpful during the builds logs troubleshooting. However, I think it's easier to reason if we rephrase like: "workspace does not exists, creating as `create-workspace` param **is set to true**". Just a minor thing, but I think it will be easier to read when you are trying to troubleshoot something.